RecipeApp/Modules/RecipeBook/ViewModel/RecipeBookViewModel.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  RecipeBookViewModel.swift
+//  RecipeApp
+//
+//  Created by Hadley Igoe on 2019-12-17.
+//  Copyright © 2019 Hadley Igoe. All rights reserved.
+//
+
+import Foundation
+
+class RecipeBookViewModel {
```

Comment:
I would really like to see a RecipeBookViewModel protocol that the viewModel implements. Can you think of a reason why I would that would be a good idea?